winml/lib/Api/LearningModelBinding.h

```diff
@@ -77,8 +77,10 @@ struct LearningModelBinding : LearningModelBindingT<LearningModelBinding, ILearn
   com_ptr<winmla::IWinMLAdapter> adapter_;
   std::vector<std::string> input_names_;
   std::vector<Ort::Value> inputs_;
+  std::vector<OrtAllocator*> input_allocators_;
```

Comment:
if we had smart pointers for these.... Ort::Allocator - as this would get us closer to where we want to be with the C_ABI as having allocators from ORT be real things and freeable.    But .. since we don't have the the C ABI yet you would have to make a smart pointer Ort::Allocator in one of our headers .. like the adapter, you can't put this in the cxx_api.h yet..

Reply:
Cool! I made an Ort::Allocator in the WinMLAdapter.h. It is based off of Base struct in onnxruntime_cxx_api.h